terminal/terminal_other.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
@derekparker, Yeah, I was just pointing out that it didn't work for @alexbrainman, because the file should be ```terminal_linux.go``` instead of ```terminal_other.go```. That is the convention used elsewhere instead of ```// +build !windows```.